src/cljs/main/broadfcui/common/sync.cljs

```diff
@@ -0,0 +1,107 @@
+(ns broadfcui.common.sync
+  (:require
+   [dmohs.react :as react]
+   [clojure.string :as string]
+   [broadfcui.common.components :as comps]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.style :as style]
+   [broadfcui.components.modals :as modals]
+   [broadfcui.endpoints :as endpoints]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(defn get-method-display [{:keys [namespace name snapshotId]}]
+  (str namespace "/" name " snapshot " snapshotId))
+
+
```

Comment:
Having a file called sync in common may be confusing as well as a component called SynchronizeModal. Maybe something like SynchronizeMethodPermissions and maybe create a package under common called methods?